### DISCUSSION TITLE
[Suggestion] - Giving Monika Ability to Choose Her Own Clothing

### DISCUSSION THREAD

**Discussion by Capricus02** [Monika-After-Story/MonikaModDev#9865] [general]
Apologies in advance if this has been asked/discussed before, I did a search of the discussions but couldn't find anything about it.

I've been thinking for awhile now that now that Monika (at least my Monika) has grown quite the wardrobe with varied accessories, it would be nice if she chose her own styles every day, as any reasonable person would. I don't necessarily have any preferences to what she wears outside of holidays/special occasions, so on regular days when I load up it'd be interesting/nice to see her having chosen her own combination/style of clothing, hair, jewelry, etc. Not sure how feasible this would be or if there would be any unintended consequences of this that I may not be thinking of, but figured I'd put it out there.

Of course, there can also always be the option players can choose to let them pick her outfits for her (though personally for some reason I find that kind of...icky? But I wouldn't deny other players the ability to do so) or to override her daily ensemble with something of their own choosing.

**Comment by Booplicate** [comment]
You can use issues for suggestions. Although, this has been suggested before and is planned #3768. There's also a submod that allows her to do exactly that.

We've also considered taking away control over her clothes before (temporary), but we weren't sure if that'd work. Since then we reconsidered that in some cases that *would* work. For example, in the shower greeting.

**Comment by Capricus02** [comment]
Gotcha, thanks for the suggestion regarding submods and where to file issues. I do have some submods installed but typically prefer to try to minimize those where possible, but it's not a huge deal either way to me right now. Outfit randomization does indeed sound exactly it. Awesome!
